tests/integration/zenodo_test.py

```diff
@@ -239,3 +242,20 @@ async def identity(x):
     v2 = await orchestrator.run()
     v2_refreshed = await orchestrator.depositor.get_record(v2.id_)
     verify_files(test_files["updated"], v2_refreshed)
+
+    # no updates to make, should not leave the conceptdoi pointing at a draft
+    v2_not_updated = await orchestrator.run()
+    # unfortunately, it looks like Zenodo doesn't propagate deletion instantly - retry this a few times.
+    latest_for_conceptdoi = await retry_async(
+        orchestrator.depositor.get_deposition,
+        args=[str(v2_not_updated.conceptdoi)],
+        kwargs={"published_only": True},
+        retry_on=(
+            ZenodoClientException,
+            aiohttp.ClientError,
+            asyncio.TimeoutError,
+            IndexError,
+        ),
+        retry_base_s=0.5,
```

Comment:
On the retry logic, in some previous logging it looked like the exponential backoff was going up to about a 1 minute wait time. I've definitely experienced network connectivity issues with some of the federal sites that last for longer than that, and wonder if it might make sense to make it possible to back off to as much as 15 or 30 minutes in the case of the scheduled periodic archiving runs that will presumably run overnight and can take a couple of hours without causing any problems. Though maybe keep it fast in the case of the CI, which we want to complete quickly.